style.css

```diff
@@ -0,0 +1,103 @@
+body {
+  padding: 0;
+  font-family: 'Inter', sans-serif;
+}
+
+.header {
+  margin-top: 10px;
+}
+
+.nav {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 15px 15px;
+}
+
+.brand {
+  width: 90px;
+  color: #28352f;
+  font-size: 20px;
+  text-align: center;
+  line-height: 28px;
+  height: 28px;
+  font-weight: 600;
+  font-style: normal;
+}
+
+a {
+  text-decoration: none;
+  color: #28352f;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  background-image: url(/header-illsutration-mobile.svg);
```

Comment:
- [ ] Kindly make sure to add background image properly. It's not displayed on your design. Fix this.
- Your design:
<img width="356" alt="Screenshot 2022-03-16 at 10 08 42 PM" src="https://user-images.githubusercontent.com/70269507/158674304-cb993cdc-571a-4cc7-ad6f-7f7414d4e448.png">
- Figma design:
<img width="294" alt="Screenshot 2022-03-16 at 10 07 33 PM" src="https://user-images.githubusercontent.com/70269507/158674396-b9f6db23-2406-4665-b995-79247943d3fa.png">